support/refc/casts.c

```diff
@@ -49,7 +49,11 @@ Value *cast_Int8_to_Integer(Value *input)
     Value_Int8 *from = (Value_Int8 *)input;
 
     Value_Integer *retVal = makeInteger();
+#ifdef INTEGER_USE_GMP
```

Comment:
Yeah, that's why I asked you in discord several days ago :) I wrote it that way because I haven't modified Makefile etc, it's easier for me to write this way. I can refactor all these ifdefs.

Reply:
Ah, I see the confusion. When I said "Both approaches are reasonable", I meant `BIGNUM_LIBBF` vs `BIGNUM_PROVIDER == LIBBF`, rather than `BIGNUM_GMP` vs `BIGNUM_LIBBF`. Your conclusion in that Discord thread about the ordering of the `if`-`else` clauses was correct (most of the time).